test/test.ts

```diff
@@ -14,6 +14,6 @@ const restoreSandbox: () => void = sandbox.restore.bind(sandbox) as () => void;
 
```

Comment:
it's independent of this change but what about renaming this file to `sandbox.ts`?

Reply:
We established that pattern of "test.js" files, but it's propably a bad name. I will fix that.